values-GCP-4.12.yaml

```diff
@@ -0,0 +1,3 @@
+cloudProvider:
```

Comment:
This is the same as values-GCP.yaml. If that is not a mistake we can probably drop values-GCP-4.12.yaml ?

Reply:
storageclass for gcp in 4.11 and 4.12 should be `standard-csi` - i will update shortly

Reply:
@mbaldessari  ?? ^^

Reply:
Patch does not apply anymore, so it should be rebased in any case. @day0hero were you going to take a look?